src/Database/Document.php

```diff
@@ -107,20 +141,19 @@ public function getUpdatedAt(): ?string
 
     /**
      * Get Document Attributes
-     * 
+     *
      * @return array
      */
     public function getAttributes(): array
     {
         $attributes = [];
 
         foreach ($this as $attribute => $value) {
-            if(array_key_exists($attribute, [
-                '$id' => true,
+            if (\array_key_exists($attribute, [
+                '$id' => ID::custom(true),
```

Comment:
```suggestion
                '$id' => true,
```

I think this should stay just true.